### PR TITLE
[NUI] Add Apis of Webview Settings.

### PR DESCRIPTION
**Pull request by AchoWang**
### Description of Change ###
Add APIs of WebView Settings, the link of csharp-binder patch as follow:

https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/254348/

### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by dongsug-song** [comment]
@AchoWang 
Hello, 
this has the csharp-binder pair patch.
please share the csharp-binder patch here and charp-binder patch should be merged before this patch is merged.
otherwise, there will be a tct-test fails which blocks the SR.
please check this.
thank you.

File: src/Tizen.NUI/src/internal/WebView/WebSettings.cs [hunk 1]
```diff
@@ -102,6 +105,56 @@ public bool EnableWebSecurity
             }
         }
 
+        /// <summary>
+        /// Enable cache builer
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool EnableCacheBuilder
```
Comment (huayongxu) [inline src/Tizen.NUI/src/internal/WebView/WebSettings.cs:112]: According to comments from https://code.sec.samsung.net/confluence/display/GFX/WebView ,
Naming rule of properties is like xxxEnabled.

File: src/Tizen.NUI/src/internal/WebView/WebSettings.cs [hunk 2]
```diff
@@ -102,6 +105,56 @@ public bool EnableWebSecurity
...
+            {
+                enableCacheBuilder = value;
+                Interop.WebSettings.EnableCacheBuilder(SwigCPtr, value);
+            }
+        }
+
+        /// <summary>
+        /// Enable do not track
+        /// </summary>
+        public bool EnableDoNotTrack
```
Comment (huayongxu) [inline src/Tizen.NUI/src/internal/WebView/WebSettings.cs:128]: EnableDoNotTrack --> DoNotTrack?

File: src/Tizen.NUI/src/internal/WebView/WebSettings.cs (hunk 1 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/internal/WebView/WebSettings.cs]: Done

File: src/Tizen.NUI/src/internal/WebView/WebSettings.cs (hunk 2 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/internal/WebView/WebSettings.cs]: Yes, It's a web standard.

**Comment by AchoWang** [comment]
> @AchoWang
> Hello,
> this has the csharp-binder pair patch.
> please share the csharp-binder patch here and charp-binder patch should be merged before this patch is merged.
> otherwise, there will be a tct-test fails which blocks the SR.
> please check this.
> thank you.

hi, Mr. Song, 

the patch's link is :
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/254348/

**Review by huayongxu — APPROVED** [review]
lgtm.

File: src/Tizen.NUI/src/internal/WebView/WebSettings.cs [hunk 3]
```diff
@@ -102,6 +105,56 @@ public bool EnableWebSecurity
...
+            }
+            set 
+            {
+                enableCacheBuilder = value;
+                Interop.WebSettings.EnableCacheBuilder(SwigCPtr, value);
+            }
+        }
+
+        /// <summary>
+        /// Enable do not track
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/internal/WebView/WebSettings.cs:126]: What is `Do not track`?

Comment (AchoWang) [reply src/Tizen.NUI/src/internal/WebView/WebSettings.cs:126]: "Do not track " is a terminology in web area,  use to protect user privacy security.

Comment (rabbitfor) [reply src/Tizen.NUI/src/internal/WebView/WebSettings.cs:126]: If so can you add the description in the summary?

Comment (AchoWang) [reply src/Tizen.NUI/src/internal/WebView/WebSettings.cs]: Done

**Pull request merged**